### PR TITLE
[MAINTENANCE] Add slow pytest marker to config and sort them alphabetically.

### PR DESCRIPTION
**Pull request by billdirks**
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -98,13 +98,14 @@ filterwarnings = [
...
+    "cloud: mark test as being relevant to Great Expectations Cloud.",
     "docs: mark a test as a docs test.",
-    "unit: mark a test as a unit test.",
-    "integration: mark test as an integration test.",
     "e2e: mark test as an E2E test.",
     "external_sqldialect: mark test as requiring install of an external sql dialect.",
-    "cloud: mark test as being relevant to Great Expectations Cloud.",
-    "base_data_context: mark test as being relevant to BaseDataContext, which will be removed during refactor",
+    "integration: mark test as an integration test.",
+    "slow: mark tests taking longer than our slow threshold.",
```
Comment (NathanFarmer) [inline pyproject.toml]: Where can the slow threshold be found?

**Review by NathanFarmer — COMMENTED** [review]
One question!

File: pyproject.toml (hunk 1 above)
Comment (cdkini) [reply pyproject.toml]: it's set to 1.0 sec right now but not enforced anywhere in the OSS repo - I think we should add it here

Comment (billdirks) [reply pyproject.toml]: @cdkini Where is it set to 1 sec? What do you mean to add it here? As a text string or as a variable so it can be used somewhere?

Comment (billdirks) [reply pyproject.toml:107]: Is it this which is 2 secs?
https://github.com/great-expectations/great_expectations/blob/bbd0795358c0817065ad3e3fe5ed6f920b1d83a8/tasks.py#L210

**Review by NathanFarmer — APPROVED** [review]
LGTM!

**Pull request merged**